lib/MessageBuilder.test.ts

```diff
@@ -0,0 +1,274 @@
+import * as zod from 'zod';
+
+import { createMessageBuilder } from './MessageBuilder.ts';
+import { isZodErrorLike } from './isZodErrorLike.ts';
+import { isNonEmptyArray } from './utils/NonEmptyArray.ts';
+
+describe('messageBuilder', () => {
```

Comment:
[question] Do you think we should match the case of the file?

```suggestion
describe('MessageBuilder', () => {
```